model/src/main/kotlin/PackageCuration.kt

```diff
@@ -58,12 +57,7 @@ data class PackageCuration(
      * package with the given [identifier][pkgId].
      */
     private fun isApplicableIvyVersion(pkgId: Identifier) =
-        try {
-            val pkgIvyVersion = Semver(pkgId.version, Semver.SemverType.IVY)
-            pkgIvyVersion.satisfies(id.version)
-        } catch (e: SemverException) {
-            false
-        }
+        runCatching { Requirement.buildIvy(id.version) }.getOrNull()?.isSatisfiedBy(pkgId.version) ?: false
```

Comment:
But actually @fviernau, `isSatisfiedBy()` might throw a `SemverException`, too. And don't we also want to just return `false` in that case?

Reply:
I've changed the implementation accordingly yet again.

Reply:
I had commented exactly that on the commit you temporarily (accidentally) had yesterday on this PR.

For this commit I had considered it out of scope, but if you do it now good.

Reply:
So, good to approve?